stl/inc/exception

```diff
@@ -35,8 +35,8 @@ _STD_END
 
 _STD_BEGIN
 
-_EXPORT_STD class exception;
-_EXPORT_STD class bad_exception;
+_EXPORT_STD using exception     = exception;
+_EXPORT_STD using bad_exception = bad_exception;

```

Comment:
I think we need a comment explaining this bit of cleverness to future us. How about "We can't export types that VCRuntime has already declared, but we can export identically-named aliases."?